package.json

```diff
@@ -1,34 +1,32 @@
 {
   "name": "@bbc/psammead",
-  "version": "2.4.9",
+  "version": "3.0.0",
   "description": "Core Components Library Developed & Maintained By The Articles and Reach & Languages Team",
   "main": "index.js",
   "private": true,
-  "resolutions": {
-    "dot-prop": "4.2.1",
-    "prismjs": "1.21.0"
-  },
+  "workspaces": [
+    "packages/components/*",
+    "packages/containers/*",
+    "packages/utilities/*"
+  ],
   "scripts": {
-    "build": "npm run build:cjs && npm run build:esm",
+    "build": "yarn run build:cjs && yarn run build:esm",
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:storybook": "build-storybook -o storybook_dist -s .storybook/static",
     "bumpChangelogs": "node scripts/bumpChangelogs/cli",
     "bumpPackages": "node scripts/bumpPackages/cli",
     "changeScanner": "node scripts/changeScanner",
-    "ci:packages": "npm ci && lerna exec -- npm ci",
+    "ci:packages": "yarn install --frozen-lockfile",
```

Comment:
Lerna exec allowed us to run the 'npm ci' command in each package directory. 

This should probably be 'yarn workspaces install'

https://classic.yarnpkg.com/en/docs/cli/workspaces/#toc-yarn-workspaces-run